library/core/src/primitive_docs.rs

```diff
@@ -291,7 +291,7 @@ mod prim_never {}
 /// Surrogate code points, used by UTF-16, are in the range 0xD800 to 0xDFFF.
 ///
 /// No `char` may be constructed, whether as a literal or at runtime, that is not a
-/// Unicode scalar value:
+/// Unicode scalar value. Violating this rule causes Undefined Behavior.
```

Comment:
```
$ rg "undefined behaviour" library/ | wc -l
13
$ rg "Undefined Behaviour" library/ | wc -l
4
```
Is it really capitalized?

Reply:
:shrug: no strong opinion either way. It is a Defined Term, not just some random English noun, but as you say  we don't consistently capitalize those.